source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -174,4 +178,10 @@ OpenGLContext: class extends GpuContext {
 		fence sync()
 		(result, fence)
 	}
+	createMesh: override func (positions: FloatPoint3D[], textureCoordinates: FloatPoint2D[]) {
```

Comment:
I think in the context of computer graphics yes, *points* would probably create more confusion than clarity. Maybe *vertices* instead?